app/src/main/kotlin/org/fossify/keyboard/helpers/Config.kt

```diff
@@ -51,6 +51,90 @@ class Config(context: Context) : BaseConfig(context) {
+
+    var languageGreekSelected: Boolean
+        get() =  prefs.getBoolean(LANGUAGE_GREEK_SELECTED, false)
+        set(languageGreekSelected) = prefs.edit().putBoolean(LANGUAGE_GREEK_SELECTED, languageGreekSelected).apply()
+
+    var languageLithuanianSelected: Boolean
+        get() =  prefs.getBoolean(LANGUAGE_LITHUANIAN_SELECTED, false)
+        set(languageLithuanianSelected) = prefs.edit().putBoolean(LANGUAGE_LITHUANIAN_SELECTED, languageLithuanianSelected).apply()
+
+    var languageNorwegianSelected: Boolean
+        get() =  prefs.getBoolean(LANGUAGE_NORWEGIAN_SELECTED, false)
+        set(languageNorwegianSelected) = prefs.edit().putBoolean(LANGUAGE_NORWEGIAN_SELECTED, languageNorwegianSelected).apply()
+
+    var languagePolishSelected: Boolean
+        get() =  prefs.getBoolean(LANGUAGE_POLISH_SELECTED, false)
+        set(languagePolishSelected) = prefs.edit().putBoolean(LANGUAGE_POLISH_SELECTED, languagePolishSelected).apply()
+
+    var languageRomanianSelected: Boolean
+        get() =  prefs.getBoolean(LANGUAGE_ROMANIAN_SELECTED, false)
+        set(languageRomanianSelected) = prefs.edit().putBoolean(LANGUAGE_ROMANIAN_SELECTED, languageRomanianSelected).apply()
+
+    var languageRussianSelected: Boolean
+        get() =  prefs.getBoolean(LANGUAGE_RUSSIAN_SELECTED, false)
+        set(languageRussianSelected) = prefs.edit().putBoolean(LANGUAGE_RUSSIAN_SELECTED, languageRussianSelected).apply()
+
+    var languageSlovenianSelected: Boolean
+        get() =  prefs.getBoolean(LANGUAGE_SLOVENIAN_SELECTED, false)
+        set(languageSlovenianSelected) = prefs.edit().putBoolean(LANGUAGE_SLOVENIAN_SELECTED, languageSlovenianSelected).apply()
+
+    var languageSpanishSelected: Boolean
+        get() =  prefs.getBoolean(LANGUAGE_SPANISH_SELECTED, false)
+        set(languageSpanishSelected) = prefs.edit().putBoolean(LANGUAGE_SPANISH_SELECTED, languageSpanishSelected).apply()
+
+    var languageSwedishSelected: Boolean
+        get() =  prefs.getBoolean(LANGUAGE_SWEDISH_SELECTED, false)
+        set(languageSwedishSelected) = prefs.edit().putBoolean(LANGUAGE_SWEDISH_SELECTED, languageSwedishSelected).apply()
+
+    var languageTurkishQSelected: Boolean
+        get() =  prefs.getBoolean(LANGUAGE_TURKISH_Q_SELECTED, false)
+        set(languageTurkishQSelected) = prefs.edit().putBoolean(LANGUAGE_TURKISH_Q_SELECTED, languageTurkishQSelected).apply()
+
+    var languageUkrainianSelected: Boolean
+        get() =  prefs.getBoolean(LANGUAGE_UKRAINIAN_SELECTED, false)
+        set(languageUkrainianSelected) = prefs.edit().putBoolean(LANGUAGE_UKRAINIAN_SELECTED, languageUkrainianSelected).apply()
+
+    var languageVietnameseTelexSelected: Boolean
+        get() =  prefs.getBoolean(LANGUAGE_VIETNAMESE_TELEX_SELECTED, false)
+        set(languageVietnameseTelexSelected) = prefs.edit().putBoolean(LANGUAGE_VIETNAMESE_TELEX_SELECTED, languageVietnameseTelexSelected).apply()
+
```

Comment:
I was just passing by and noticed this, why not just save the selected languages to a set preference?

Example: https://github.com/FossifyOrg/Music-Player/blob/60688c8f378b6fc75875181fb4142f31c1132f77/app/src/main/kotlin/org/fossify/musicplayer/helpers/Config.kt#L122-L124

Reply:
@naveensingh I fixed this, switched to using a mutable set instead of a boolean value for each language.